topics/metabolomics/tutorials/lcms/tutorial.md

```diff
@@ -0,0 +1,1150 @@
+contributors:
+- melpetera
+- lecorguille
+- jfrancoismartin
+- yguitton
+- workflow4metabolomics
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+You may already know that there are different types of omic sciences; metabolomics is the closest to phenotypes. 
+It studies different types of matrices, as blood, urines, tissues, including plants of course, and focus on studying really small molecules 
+which are called metabolites, to better understand matters linked to the metabolism. However, studying metabolites is not a piece of cake 
+since it requires several critical steps which still have some major bottlenecks. Metabolomics is still quite a young science, and has many 
+kinds of specific matters. 
+
+One of the three main technologies used to perform metabolomic analysis is Liquid-Chromatography Mass Spectrometry (LC-MS). Data analysis 
+for this technology requires a large variety of steps, going from the way to extract information from your raw data to statistical analysis 
+and annotation. To be able to perform a complete LC-MS analysis in a single environment, the [Wokflow4Metabolomics](http://workflow4metabolomics.org/) 
+team profides Galaxy modules dedicated to metabolomics. Here is a tutorial aiming at explaining what the main steps of untargetted LC-MS data processing 
+for metabolomic analysis are, and how to conduct metabolomic data analysis from preprocessing to annotation using Galaxy. 
+
+To illustrate this approach, we will use data from {% cite Thvenot2015 %}. The objectives of this paper was to analyze
+the inﬂuence of age, body mass index, and gender on the urine metabolome. To do so, the authors collected samples
+from 183 employees from the French Alternative Energies and Atomic Energy Commission (CEA) and performed LC-HRMS LTQ-Orbitrap
+(negative ionization mode) (**TODO** explain the terms).
+
+Since the original dataset takes a few hours to be processed, we chose to take a limited subset of individuals for this tutorial.
+This will allow you to perform an example of metabolomic workflow from pre-processing to annotation in a limited time, even though
+the results obtained may not be reliable from a scientific point of view due to a sample size way too small. Nevertheless,
+the chosen diversity of sample will allow you to explore the bases of a metabolomic workflow.
+
+We chose a subset of 9 samples, composed of 6 biological samples and 3 quality-control pooled samples (QC pools - mix of all
+biological samples).
+
+To analyze these data, we will then follow a light version of the [LC-MS workflow](http://workflow4metabolomics.org/the-lc-ms-workflow),
+developed by the [Wokflow4metabolomics group](http://workflow4metabolomics.org/), ({% cite Giacomoni2014 %}, {% cite Guitton2017 %}).
+It is composed of 4 main parts: the preprocessing part that extracts ions from raw data, the data processing part that checks the quality of data 
```

Comment:
Done but the picture is maybe too small.
Is there a way to have a clickable picture to have it in full size?

Reply:
Thanks :)

Reply:
I've hyperlinked it to the image itself, so they can view the full size image by clicking on it (maybe some day we can edit the image so that the 4 boxes on the right are below the other box so that it's more readable within the tutorial as well)

Reply:
I like the left-to-right workflow.
But yes, I will do the Japanese workflow style 😛

Reply:
haha :P I am also fine with leaving it like this, your choice, but no easy way to zoom within the tutorial page itself

Reply:
Done, thanks for your advices

Reply:
awesome!